person.rb

```diff
@@ -0,0 +1,31 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @id = Random.rand(1..1000)
+  end
+
+  # getter for id
+  attr_reader :id
+
+  # getter for name
+  attr_accessor :name
+
+  # getter for age
+  attr_accessor :age
+
+  # setter for name
+
+  # setter for age
```

Comment:
- [ ] [OPTIONAL] Please notice that you can write the getters and setters both by using attr_accessor, so there is no need for additional empty setter comments `  # setter for name  # setter for age`
- [ ] [OPTIONAL] Also you can create attr_accessors for multiple methods in a single line like :point_down: 
      `attr_accessor :name, :age `